torch/distributions/kl.py

```diff
@@ -174,12 +174,10 @@ def _kl_dirichlet_dirichlet(p, q):
     # From http://bariskurt.com/kullback-leibler-divergence-between-two-dirichlet-and-beta-distributions/
     sum_p_alpha = p.alpha.sum(-1)
     sum_q_alpha = q.alpha.sum(-1)
-    t1 = torch.lgamma(sum_p_alpha)
-    t2 = torch.lgamma(sum_q_alpha)
-    t3 = p.alpha.lgamma().sum(-1)
-    t4 = q.alpha.lgamma().sum(-1)
-    t5 = (p.alpha - q.alpha) * (p.alpha.digamma() - sum_p_alpha.digamma())
-    return t1 - t3 - t2 + t4 + t5.sum(-1)
+    t1 = torch.lgamma(sum_p_alpha) - p.alpha.lgamma().sum(-1)
```

Comment:
Did this accidentally leak from your other PR?

Reply:
Ah, that could have happened. I will revert those changes right away.